cwms-data-api/src/main/java/cwms/cda/api/ClobController.java

```diff
@@ -156,19 +155,27 @@ public void getAll(@NotNull Context ctx) {
             ctx.result(result);
             ctx.contentType(contentType.toString());
             requestResultSize.update(result.length());
-
         }
     }
 
 
     @OpenApi(
             queryParams = {
                     @OpenApiParam(name = OFFICE, description = "Specifies the owning office."),
+                    @OpenApiParam(name = CLOB_ID, description = "If this _query_ parameter is provided the id _path_ parameter "
+                            + "is ignored and the value of the query parameter is used.   "
+                            + "Note: this query parameter is necessary for id's that contain '/' or other special "
+                            + "characters.  Because of abuse even properly escaped '/' in url paths are blocked.  "
+                            + "When using this query parameter a valid path parameter must still be provided for the request"
```

Comment:
It's actually a slight failure in Javalin in how it even processes the paths... there's a section that just, doesn't.

But I like this wording.

Reply:
Well, and could be both.

Reply:
what could be both?

Reply:
Bad processing by Javalin (I checked)
But also the security issue. I know that encoding in the path can cause issues, but even after I turned off that check it still doesn't work even though the manual says it should.